#camera_matrix: [770.575867, 0.000000,   319.001556, 0.000000
 #               ,0.000000,  770.891785, 238.297930, 0.000000
  #              ,0.000000,  0.000000,   1.000000,   0.000000]

camera_matrix: [640.0,      0.000000,   320.0, 0.000000
                ,0.000000,  640.0,      240.0, 0.000000
                ,0.000000,  0.000000,   1.000000,   0.000000]

rdc: [0.0,         0.0,    0.0000,      
      -1.0,    0.0000,         -1.0,        
   0.0,         1.0,         0.0    ]

tdc: [ 0.0, 0.0, 0.18]




